test/utils.py

```diff
@@ -51,11 +51,12 @@ def download_file(url, target):
 
 
 def LONG_TESTS():
-    LONG_TESTS = 0
     # override LONG_TESTS with environment variable if available
     if os.getenv("LONG_TESTS"):
         LONG_TESTS = int(os.getenv("LONG_TESTS"))
-    return LONG_TESTS
+        if LONG_TESTS > 0:
+            return True
+    return False
```

Comment:
The function is still untyped, should be typed as returning `bool`.

Also, type checkers are still unhappy with this implementation (both mypy and pyright). They are not smart enough to determine that `os.getenv("LONG_TESTS")` is not None or an empty string inside the `if` statement because you didn't store the value in a variable.

`LONG_TESTS` variable inside the function is a bad choice, only constants should be in capital letters.

Consider this implementation:
```py
def LONG_TESTS() -> bool:
    # override LONG_TESTS with environment variable if available
    env_var = os.getenv("LONG_TESTS")
    if env_var and int(env_var) > 0:
        return True
    return False
```

However, I'm still curious as to when this env variable can be anything other than "0" or "1"?